content/docs/tooling.md

```diff
@@ -27,10 +27,12 @@ The language you're looking for is not here? You have created a new code generat
 
 The following is a list of tools that generate human-readable documentation from an AsyncAPI document.
 
-| Link           | Description    |
-| :------------- | :------------- |
-| [AsyncAPI Generator](https://github.com/asyncapi/generator) | Use your AsyncAPI definition to generate literally anything. Markdown documentation, Node.js code, HTML documentation, anything!
-| [Widdershins](https://github.com/Mermade/widdershins) | OpenApi 3.0 / Swagger 2.0 / AsyncAPI 1.0 definition to Slate / Shins compatible markdown. Thanks to [@PermittedSoc](https://twitter.com/@Permittedsoc).
+| Link           | Description    | Language/Kind |
+| :------------- | :------------- | :------------ |
+| [AsyncAPI Generator](https://github.com/asyncapi/generator) | Use your AsyncAPI definition to generate literally anything. Markdown documentation, Node.js code, HTML documentation, anything! | Javascript
+| [Widdershins](https://github.com/Mermade/widdershins) | OpenApi 3.0 / Swagger 2.0 / AsyncAPI 1.0 definition to Slate / Shins compatible markdown. Thanks to [@PermittedSoc](https://twitter.com/@Permittedsoc). | Javascript
+| [Bump](https://bump.sh) | OpenApi 2 & 3 / AsyncAPI 2 documentation generator, with automatic changelog an visual diff. | SaaS
```

Comment:
Typo: " ...an**d** visual diff"

Reply:
Oops, sorry! Thanks for the catch, this is fixed now.